### DISCUSSION TITLE
VP18C Fan mode

### DISCUSSION THREAD

**Discussion by AhmoBecks** [veista/nilan#155] [General]
Any ideas what could cause fan modes 3 and 4 to not always be accessible? The fan speed indeed changes on the panel, but the fan's speed won't actually change. If I set fan mode to 0 from HA (or turn the machine off from the panel) and then set fan mode to 3 or 4, it runs in mode 3 or 4 for a while and then automatically changes back to fan mode 2. This is not a new issue, but it's getting annoying.

VP18C
2.37.c
Modbus 11

**Comment by veista** [comment]
You are best served by your local Nilan support. I am not that, nor is this repo.

My suggestion is, you disable all automatic fan adjustments, for example outdoor temperature, humity and co2.

**Reply by AhmoBecks** [reply]
I never meant you to be local Nilan support. My best guess is that this is related to modbus. I guess i figure it out.

**Reply by veista** [reply]
Sorry if I was too blunt. It does not sound like it is a problem with the integration or modbus. It sounds like you need support from Nilan on how to operate your device.

Automatic functions like humidity control override manual fan adjustments.

**Reply by AhmoBecks** [reply]
It's fine. You were helpfull and right. After i disabled humidity control everything is working as it should.
